bms-monitor.js

```diff
@@ -1,18 +1,90 @@
+const warningSettings ={
+  temperature: true,
+  soc: true,
+  chargeRate: true,
+};
+
+let language='en';
+
+const messages={
+  en:{
+    tempLow:'Warning: Approaching lower temperature limit',
+    tempHigh:'Warning: Approaching upper temperature limit',
+    socLow:'Warning: Approaching discharge',
+    socHigh: 'Warning: Approaching charge-peak',
+    chargeRateLow:'Warning: Approaching lower chargeRate limit',
+    chargerateHigh:'Warning: Approaching upper chargeRate limit',
+    tempOut:'Temperature is out of range!',
+    socOut:'State of Charge is out of range!',
+    chargeRateOut:'Charge rate is out of range!'
+
+  },
+
+  ge:
+  {
+    tempLow:'Warnung: Untere Temperaturgrenze nähert sich',
+    tempHigh:'Warnung: Die Temperaturobergrenze nähert sich',
+    socLow:'Warnung: Entladung steht kurz bevor',
+    socHigh:'Warnung: Ladespitze naht',
+    chargeRateLow:'Warnung: Untere Laderatengrenze nähert sich',
```

Comment:
Good translation mapping :+1: 
But the translations are not tested. Consider this sequence: someone asks for a new parameter, we add it. The messages get added in the `en:` part but the developer forgets to add in the `ge:` object. The tests will still pass. To test the entire thing, we have to simulate each and every message in every language we support.
Think of a test that can expose this mistake (missing translation) without having to do every test in every language.